src/pages/EmployeeProfile/components/style.css

```diff
@@ -0,0 +1,33 @@
+h2{
+    text-align: center;
+}
+.component{
+    border-radius: 5px;
+    margin: 1vh;
+    height: 100%;
+    position: relative;
+}
+.banner{
+    height: 15vh;
+}
+.banner-paper{
+    height: 150%;
+}
+.list-item-text{
+    text-align: center;
+}
+.about-text{
+    padding: 5vh;
+}
+.form-control{
+    padding: 1vh;
+    height: fit-content;
+    width: 95%;
+    position: absolute;
+    top: 50%; left: 50%;
+    transform: translate(-50%,-50%);
+}
+.contact{
+    padding: 1vh;
+    height: fit-content;
+}
```

Comment:
You can style the most components directly, because you use the classed only once, like the "about-text" class
Use the sx Tag for this